test/DoctrineORMModuleTest/Console/Helper/MigrationsConfigurationHelperTest.php

```diff
@@ -0,0 +1,136 @@
+        $inputDefinition = new InputDefinition([$inputOption]);
+        $request         = new Request([]);
+        $requestInput    = new RequestInput($request, $inputDefinition);
+        $helper          = new MigrationsConfigurationHelper($this->serviceLocator);
+
+        $configuration = $helper->getMigrationConfig($requestInput);
+
+        $this->assertSame(
+            $this->serviceLocator->get('doctrine.migrations_configuration.orm_default'),
+            $configuration
+        );
+    }
+
+    /**
+     * This test queries for a non-orm_default object manager
+     * and throws an exception when the configuration is not found
+     */
+    public function testGetNonDefaultMigrationConfig(): void
+    {
+        $inputOption     = new InputOption('object-manager', [], InputOption::VALUE_REQUIRED);
+        $inputDefinition = new InputDefinition([$inputOption]);
+        $request         = new Request([
+            'index.php',
+            '--object-manager=doctrine.entitymanager.orm_some_other_name',
+        ]);
+        $requestInput    = new RequestInput($request, $inputDefinition);
+
+        $this->expectException(ServiceNotFoundException::class);
+        $this->expectExceptionMessage(
+            'Unable to resolve service '
+                . '"doctrine.migrations_configuration.orm_some_other_name" to '
+                . 'a factory; are you certain you provided it during configuration?'
+        );
+        (new MigrationsConfigurationHelper($this->serviceLocator))->getMigrationConfig($requestInput);
+    }
+
+    /**
+     * This test ensures the object manager is valid using the regex.  However this is only an
+     * issue in theory.  In practice an invalid object manager will be caught in the CliConfigurator
+     * before it ever gets to the helper.
+     */
+    public function testInvalidObjectManagerAlias(): void
+    {
+        $inputOption     = new InputOption('object-manager', [], InputOption::VALUE_REQUIRED);
+        $inputDefinition = new InputDefinition([$inputOption]);
+        $request         = new Request([
+            'index.php',
+            '--object-manager=doctrine.entitymanagerorm_some_other_name',
+        ]);
+        $requestInput    = new RequestInput($request, $inputDefinition);
+
+        $this->expectException(RuntimeException::class);
+        $this->expectExceptionMessage(
+            'The object manager alias is invalid: doctrine.entitymanagerorm_some_other_name'
+        );
+        (new MigrationsConfigurationHelper($this->serviceLocator))->getMigrationConfig($requestInput);
+    }
+
+    public function testValidObjectManagerAlias(): void
+    {
+        $inputOption     = new InputOption('object-manager', [], InputOption::VALUE_REQUIRED);
+        $inputDefinition = new InputDefinition([$inputOption]);
+        $request         = new Request([
+            'index.php',
+            '--object-manager=doctrine.entitymanager.orm_default',
```

Comment:
Can you please use a different id than the default manager? So that this test differs from the one for the default manager in case of regressions.

Reply:
Done and I removed the last commit and put those changes into the new commit for the second object manager.